velox/exec/tests/TaskTest.cpp

```diff
@@ -71,6 +71,9 @@ class TaskTest : public HiveConnectorTestBase {
 
     return {task, results};
   }
+
+  std::unique_ptr<folly::Executor> driverExecutor_{
```

Comment:
I see that before this change we used executor_ from HiveConnectorTestBase:

```
void HiveConnectorTestBase::SetUp() {
  OperatorTestBase::SetUp();
  executor_ = std::make_unique<folly::IOThreadPoolExecutor>(3);
```

That executor is still used in other places, no? Should we fix the other places as well?

Do we need 2 executors: one for spill and prefetch (IO) and another one for the driver (CPU)?

Would it make sense to define these 2 executors in the base class using descriptive names and update the callers?